src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -1480,6 +1480,9 @@
   - name: --max-surge
     type: string
     short-summary: Extra nodes used to speed upgrade. When specified, it represents the number or percent used, eg. 5 or 33%
+  - name: --drain-timeout
+    type: string
```

Comment:
```suggestion
    type: int
```
seems to be supported here, right? See `--max-pods`?